seshdash/sesh-ui/js/sesh_dashboard.js

```diff
@@ -87,7 +87,7 @@ $('#containerhigh').highcharts({
                 }
             },
             labels: {
-                format: '{value} Wh',
+                format: '{value} HW',
```

Comment:
can you modify this to be Wh? I think this was changed any you merged it back somehow.